### PR TITLE
chore(electron): Move off deprecated remote module

### PR DESCRIPTION
**Pull request by bmulholland**
This is [a requirement for electron v14, which drops the module](https://www.electronjs.org/blog/electron-14-0/#removed-remote-module). We can do it before the upgrade, for smaller changes.

### CODE REVIEW

**Comment by bmulholland** [comment]
Current dependencies on/for this PR:
* main
  * **PR #613** <a href="https://app.graphite.dev/github/pr/gitify-app/gitify/613" target="_blank">Graphite</a> 
    * **PR #649** <a href="https://app.graphite.dev/github/pr/gitify-app/gitify/649" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/gitify-app/gitify/649?utm_source=stack-comment).

**Comment by bmulholland** [comment]
This introduces a new issue, probably on a check for isLinux: `[79860:1003/134934.778674:INFO:CONSOLE(2)] "Uncaught TypeError: Cannot read property 'process' of undefined"`

**Comment by bmulholland** [comment]
Fixed

File: main.js [hunk 1]
```diff
@@ -75,6 +77,9 @@ menubarApp.on('ready', () => {
       }
     }
   });
+  ipcMain.on('get-platform', (event) => {
```
Comment (bmulholland) [inline main.js:80]: We should switch to this approach for all usage of remote. See https://github.com/gitify-app/gitify/issues/650

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**